apps/indexer/lib/indexer/transform/celo_accounts.ex

```diff
@@ -40,13 +39,13 @@ defmodule Indexer.Transform.CeloAccounts do
   defp get_rates(logs, oracle_address) do
     logs
     |> Enum.filter(fn log -> log.address_hash == oracle_address end)
-    |> Enum.filter(fn log -> log.first_topic == CeloAccount.oracle_reported_event() end)
+    |> Enum.filter(fn log -> log.first_topic == Events.oracle_reported_event() end)
     |> Enum.reduce([], fn log, rates -> do_parse_rate(log, rates) end)
   end
 
   def get_names(logs) do
     logs
-    |> Enum.filter(fn log -> log.first_topic == CeloAccount.account_name_event() end)
+    |> Enum.filter(fn log -> log.first_topic == Events.account_name_event() end)
```

Comment:
There is a commented out code below that keeps a reference to `CeloAccount`. We should either remove the comment or update it.